Info card

cyra997 opened the pull request:
First version of the info card with header and text has been made.

cyra997 committed:
initial commit

jimpala commented on src/App.tsx:
```diff
@@ -9,11 +9,20 @@ const App: React.FunctionComponent = () => {
+
+      <Flex
+      flexDir="column"
+      w="250"
```
Use design system token for the width here (e.g: "sm", "lg", "3xl")

jimpala replied:
Smaller widths (like padding) use numbers in the Chakra design system theme file, larger widths use "sm", "3xl" etc

cyra997 replied:
The requested change has been made.

jimpala reviewed (changes requested):
Please implement changes to sizing given here!

cyra997 committed:
Changes to sizes
Used tokens and numbers from the Chakra design system to replace previous unstandardised values.

jimpala reviewed (approved):
Looks good to me!
